heron/common/tests/cpp/network/switch_unittest.cpp

```diff
@@ -32,7 +32,7 @@
 #include "threads/modinit.h"
 #include "network/modinit.h"
 
-static const sp_uint32 SERVER_PORT = 61000;
+static const sp_uint32 SERVER_PORT = SockUtils::getFreePort();
```

Comment:
Instead of using global variable - can you the function directly where a port is needed?